### PR TITLE
Copy editing

### PR DESCRIPTION
**Pull request by ljacqu**
This addresses some of the points I raised in #147. The first commit only contains replacements from `<code>foo</code>` to Markdown's <code>```</code> style of blocks. The former doesn't preserve newlines. 
The second commit also includes such replacements but also fixes wrong or old links and improves the grammar of certain posts.

### CODE REVIEW

**Comment by DTrejo** [comment]
@brianleroux i looked over this PR and it looks good to me. Want to rebase & merge it?

I wasn't able to view the changes locally b/c of this error w/ `npm run start`, I think:

```
/Users/dtrejo/Dropbox/dev/forks/wtfjs/node_modules/aws-sdk/lib/request.js:31
            throw err;
            ^

CredentialsError: Missing credentials in config
    at ClientRequest.<anonymous> (/Users/dtrejo/Dropbox/dev/forks/wtfjs/node_modules/aws-sdk/lib/http/node.js:83:34)
    at Object.onceWrapper (events.js:314:30)
    at emitNone (events.js:105:13)
    at ClientRequest.emit (events.js:207:7)
    at Socket.emitTimeout (_http_client.js:721:34)
    at Object.onceWrapper (events.js:314:30)
    at emitNone (events.js:105:13)
    at Socket.emit (events.js:207:7)
    at Socket._onTimeout (net.js:401:8)
    at ontimeout (timers.js:488:11)
```

**Comment by ljacqu** [comment]
Would love to hear something about this :) If there's some action here I can work on some of the other stuff I mentioned in my issue.

File: src/md/2014-02-25-array-indexof.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
 ```
 Let kPresent be the result of calling the [[HasProperty]] internal method of O with argument ToString(k).
 ```
 
-In that, `k` is a number corresponding to an array index and O is the array itself. 
+In that, `k` is a number corresponding to an array index and `O` is the array itself. 
 Since elided elements were not defined the array does not have a property for the corresponding index.
 
-*Ok. But why we get `undefined` when get first array element?*
+*Ok. But why do we get `undefined` when get the first array element?*
```
Comment (ljacqu) [inline src/md/2014-02-25-array-indexof.md]: when __getting__*

**Comment by DTrejo** [comment]
Just let him know: https://twitter.com/ddtrejo/status/902892724109549568

**Comment by brianleroux** [comment]
Sorry about that! @DTrejo you need to setup AWS per https://arc.codes/quickstart

(I can get you a user account on my personal AWS if you want to do deploys?)

**Pull request merged**